core/src/main/java/nu/yona/server/analysis/entities/Activity.java

```diff
@@ -48,6 +49,11 @@ public void setEndTime(Date endTime)
 		this.endTime = endTime;
 	}
 
+	public int getDurationMinutes()
+	{
+		return (int) TimeUnit.MILLISECONDS.toMinutes(this.getEndTime().getTime() - this.getStartTime().getTime());
```

Comment:
This should return 1 if the time difference between start and end is > 0 (in any unit), to ensure we never round something to nothing.